build/pack.c

```diff
@@ -22,6 +22,10 @@
 
 #include "debug.h"
 
+#if _OPENMP < 201511
+#define priority(x)
```

Comment:
Hmm, now that I think about it - wouldn't this be too brittle a macro? What if we define/include a function `priority()` in the future? Wouldn't this replace it?